app/models/labware_creators/partial_stamped_plate.rb

```diff
@@ -25,7 +25,7 @@ def dilutions_calculator
     # The well filter will be used to identify the parent wells to be taken forward.
     # Filters on request type, library type and state.
     def well_filter
-      @well_filter ||= WellFilterAllowingPartials.new(creator: self, request_state: 'pending')
```

Comment:
This pipeline's request are 'started' not pending. I think `started` is safe to use here as is it is also an `ongoing` request state.

Reply:
Yeah that was my thinking to. I think this is more a prevention mechanism for passed requests than anything else.